NAMESPACE

```diff
@@ -47,3 +47,53 @@ importFrom("stats", "acf", "approx", "complete.cases", "cov", "deltat",
 importFrom("utils", "count.fields", "read.table", "setTxtProgressBar",
            "txtProgressBar")
 importFrom(Rcpp, evalCpp)
+
+S3method(`*`, raw_cf)
+S3method(`+`, cf)
+S3method(`+`, raw_cf)
+S3method(`-`, cf)
+S3method(`-`, raw_cf)
+S3method(`/`, cf)
+S3method(`/`, raw_cf)
+S3method(c, cf)
+S3method(c, raw_cf)
+S3method(plot, averx)
+S3method(plot, b1fit)
+S3method(plot, bootstrapfit)
+S3method(plot, cf)
+S3method(plot, cfit)
+S3method(plot, coshfit)
+S3method(plot, effectivemass)
+S3method(plot, effmass)
+S3method(plot, gevp.amplitude)
+S3method(plot, hadronacf)
+S3method(plot, massfit)
+S3method(plot, matrixfit)
+S3method(plot, ofit)
+S3method(plot, outputdata)
+S3method(plot, pionff)
+S3method(plot, pionfit)
+S3method(plot, raw_cf)
+S3method(plot, rhofit)
+S3method(plot, uwerr)
+S3method(print, averx)
+S3method(print, bootstrapfit)
+S3method(print, cf)
+S3method(print, effectivemassfit)
+S3method(print, ofit)
+S3method(print, pionfit)
+S3method(print, raw_cf)
+S3method(summary, averx)
+S3method(summary, bootstrapfit)
+S3method(summary, cf)
+S3method(summary, coshfit)
+S3method(summary, effectivemass)
+S3method(summary, effectivemassfit)
+S3method(summary, gevp.amplitude)
+S3method(summary, hadronacf)
+S3method(summary, matrixfit)
+S3method(summary, ofit)
+S3method(summary, pionff)
+S3method(summary, pionfit)
+S3method(summary, raw_cf)
+S3method(summary, uwerr)
```

Comment:
why is this needed?

Reply:
When one does not attaches the namespace, the S3 method resolution does not work:

```
> plot(hadron::samplecf)
Fehler in xy.coords(x, y, xlabel, ylabel, log) : 
  'x' is a list, but does not have components 'x' and 'y'
```

With these explicit S3 methods, the above should work, but I have not tested it.